.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: MSI PRO Z-690A
```

Comment:
CI probably needs updating to build for both platforms

Reply:
Yup, I am still working on polishing such things

Reply:
Added a bit later: https://github.com/Dasharo/coreboot/commit/9899b427131f4fe081918cbe779fab72407527c5